index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Camilo Vela Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100&family=Roboto:wght@100&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <header class="header">
+      <a class="logo">John Doe</a>
+      <img class= "normal_button" src="images/Enabled.png" alt="normal_button">
+    </header>
+    <div class="headline">
+      <div class="primary_text">
+        <a>Hey There. I'm Camilo</a>
+      </div>
+      <div class="supporting_text">
+        <a>I'm a Software Developer</a>
+      </div>
+      <div class="description">
+        <a>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don't hestiate to contact me.</a>
+      </div>
+      <div class="social_media">
+        <img src="images/Github_logo.png" alt="Github_logo" class="social_media_logo">
+        <img src="images/Linkedin_logo.png" alt="Linkedin_logo" class="social_media_logo">
+        <img src="images/Angellist_logo.png" alt="Angellist_logo" class="social_media_logo">
+        <img src="images/Twitter_logo.png" alt="Twitter_logo" class="social_media_logo">
+        <img src="images/Medium_logo.png" alt="Medium_logo" class="social_media_logo">
+      </div>

```

Comment:
- [ ] your social media icons are not displayed please make sure to fix that also they should in wrapped them in `ul` tag and each one in `li` tag and the icon should be `<a>` tag

- Yours 

![image](https://user-images.githubusercontent.com/61790587/176684178-b91053ee-974c-42f4-8ea8-3ec7871a8551.png)